src/renderers/shared/reconciler/ReactMultiChild.js

```diff
@@ -342,7 +344,7 @@ var ReactMultiChild = {
           );
         }
         nextIndex++;
-        lastPlacedNode = ReactReconciler.getNativeNode(nextChild);
+        lastPlacedNode = ReactReconciler.getLastNativeNode(nextChild);
```

Comment:
Since text components consist of multiple nodes node, I needed a way to get a reference to the last node (the closing comment) here...